subsys/power/policy/Kconfig

```diff
@@ -1,64 +1,67 @@
-
 choice
 	prompt "Idle State Power Management Policy"
-	depends on PM_CONTROL_OS
 	help
 	  Storage back-end to be used by the settings subsystem.
 
-config PM_POLICY_RESIDENCY
+config SYS_PM_POLICY_RESIDENCY
 	bool "PM Policy based on CPU residency"
 	help
 	  Select this option for PM policy based on CPU residencies.
 
-config PM_POLICY_DUMMY
+config SYS_PM_POLICY_DUMMY
 	bool "Dummy PM Policy"
 	help
 	  Dummy PM Policy which simply returns next PM state in a loop.
 
+config SYS_PM_POLICY_APP
+	bool "Application PM Policy"
+	help
+	  When this option is selected, the application must provide PM policy.
+
 endchoice
 
-if PM_POLICY_RESIDENCY
+if SYS_PM_POLICY_RESIDENCY
 
-config PM_LPS_MIN_RES
-	int "LPS minimum residency"
-	depends on SYS_POWER_STATE_CPU_LPS_SUPPORTED
+config SYS_PM_LPS_0_MIN_RES
+	int "LPS_0 minimum residency"
```

Comment:
What about "Low Power state X minimum residency"? And then "Deep Sleep state X ..." below.

Reply:
Fixed